### ISSUE TITLE
normal.gee with Amelia imputed data

### ISSUE THREAD

**Issue by christophergandrud** [IQSS/Zelig#263]
From Fernando Bizzarro:

```r
library(Zelig)
load('~/Desktop/geeIssue//low.imput.RData')

# Within Imputs
z1 <- zelig(low_volatility ~ psi_low, data = low.imput$imputations[[1]],
      model = 'normal.gee', corstr = 'AR-M', id = 'country_id')
```

estimates the model. 

The following does not:

```r
# Across
z2 <- zelig(low_volatility ~ psi_low, data = low.imput, model = 'normal.gee',
      corstr = 'AR-M', id = 'country_id')
```

instead returning:

```r
Error in model.frame.default(formula = low_volatility ~ psi_low, data = as.data.frame(.),  : 
  variable lengths differ (found for '(id)') 
```

**Comment by christophergandrud** [comment]
It appears that geeglm accepts as its `id` argument a vector the same length as the estimation data frame. With MI data it must be using the column for all of the combined MI data set.

**Comment by christophergandrud** [comment]
Resolved. estimation, `setx`, and `sim` work.

Though `combine_coef_se` returns:

```r
Error in match.arg(type) : 
  'arg' should be one of “pearson”, “working”, “response”
```

**Comment by christophergandrud** [comment]
Use naive vcov matrix as indicated in: https://stats.stackexchange.com/a/20440

**Comment by christophergandrud** [comment]
Should be resolved in d4f5f7d3dfab03557f8d4fa246d91f03226db682
